[Droid] Show message text specified in ActionSheetConfig

alexshikov opened the pull request:
This is a fix for #425 and #250

aritchie commented:
This will unfortunately only fix the latest versions of android and breaks older versions of android by removing the list from the dialog.

alexshikov commented:
Hah, I see...
By "removing the list" you mean the case, when there is a message + a list, right?
Does it make sense to show a warning for Android users in that case? Or throw an exception...

aritchie commented:
Correct.  

Perhaps a log would help, definitely not an exception as it will cause grief for iOS.

alexshikov commented:
I've added logging a warning in the discussed case.

Don't really like the method name `ShowWarningsIfAny`. Maybe that should be renamed.

Also, I can't compile a solution from Visual Studio for Mac. Hope I've added all needed usings and semicolons.
